File: Data Structures & Algorithms/anagram-groups/submission-0.java
class Solution {
    public List<List<String>> groupAnagrams(String[] strs) {
        
  HashMap<String,List<String>> ans = new HashMap<>();
    for (String s : strs ) {
        int[] temp = new int[26];
        for (char c : s.toCharArray()) {
            temp[c -'a']++;
        }
        String pop = Arrays.toString(temp);
        if (!(ans.containsKey(pop))) {
            ans.put(pop,new ArrayList<>());
        }
        ans.get(pop).add(s);
    }
        
    return new ArrayList<>(ans.values());
    }
}

File: Data Structures & Algorithms/longest-palindromic-substring/submission-0.py
class Solution:
    def longestPalindrome(self, s: str) -> str:
        temp = -1
        stro=''
        for i in range(len(s)):
            l=i
            r=i
            while l>=0 and r<len(s) and s[l]==s[r]:
                if (r-l+1>temp):
                    temp=r-l+1
                    stro=s[l:r+1]
                l-=1
                r+=1
            l=i
            r=i+1
            while l>=0 and r<len(s) and s[l]==s[r]:
                if (r-l+1>temp):
                    temp=r-l+1
                    stro=s[l:r+1]
                l-=1
                r+=1
        return stro